src/components/03-components/09-accordion/accordion.njk

```diff
@@ -1,38 +1,35 @@
+{#
```

Comment:
Remove commented out code

Reply:
I was specifically asked to keep that in and commented out